cloud/app/routes/brains.py

```
@@ -0,0 +1,42 @@
+@router.post("/brains/connect", response_model=ConnectResponse)
+async def connect_brain(
+    body: ConnectRequest,
+    brain: dict = Depends(get_current_brain),
+) -> ConnectResponse:
+    """Register/reconnect a brain. Called by CloudClient.connect()."""
+    _log.info("Brain connected: %s (name=%s)", brain["id"], body.brain_name)
+    return ConnectResponse(brain_id=brain["id"])
+
+
+@router.get("/brains")
+async def list_brains(brain: dict = Depends(get_current_brain)) -> list[dict]:
```

Ruff (0.15.9)

[warning] Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)